dev/com.ibm.ws.security.openidconnect.client/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -105,13 +105,13 @@ jwkClientSecret=JWK client password
 jwkClientSecret.desc=Specifies the client password to include in the basic authentication scheme of the JWK request.
 
 grantType=Grant type
-grantType.desc=Specifies the grant type to use for this client.
+grantType.desc=Specifies the grant type to use for this client. Use of the responseType attribute is preferred instead.
 
 grantType.authorization_code=Authorization code grant type
 grantType.implicit=Implicit grant type
 
 responseType=Response type
-responseType.desc=Specifies the required response type for this client.
+responseType.desc=Specifies the response requested from the provider, either an authorization code or implicit flow tokens.
```

Comment:
Would probably prefer to say "Specifies the response **type** that is requested from the provider"